examples/avro-cli.py

```diff
@@ -36,19 +37,37 @@
 """)
 
 
-# User stores the deserialized user Avro record.
+"""
+    User stores the deserialized user Avro record.
+"""
+
+
 class User(object):
 
-    # Explicitly declare data members to prevent creation of dynamic properties
-    __slots__ = ["name", "favorite_number", "favorite_color"]
+    """
+        Slots are both more performant and memory efficient than the standard class __dict__.
+        This has the added benefit of adding stronger type safety by disabling dynamic attribute creation.
+        If the deserialized Avro object does not match the User class definition exactly an attribute error
+        will be raised.
+    """
+    __slots__ = ["name", "favorite_number", "favorite_color", "correlation_id"]
 
     def __init__(self, name=None, favorite_number=None, favorite_color=None):
         self.name = name
         self.favorite_number = favorite_number
         self.favorite_color = favorite_color
+        """
```

Comment:
I don't think you can a doc string for an arbitrary code line, you'd use comments for that.

Reply:
can't or won't, flake8 seemed to be okay with it and a few places mention using ''' for block comments but I agree. fixed